### ISSUE TITLE
Update code for AVLTree updateHeight()

### ISSUE THREAD

**Issue by JMCanning78** [JMCanning78/datastructures-visualization#289]
The code for the `AVLtree.__Node.updateHeight()` method changed in the textbook.  The AVLTree visualization must also change to match.  The routine should migrate from the BinaryTreeBase.py file to AVLTree.py and use the new definition:

```
      def updateHeight(self): # Update height of node from children
         self.height = max(   # Get maximum child height using 0 for
            child.height if child else 0 # empty child links
            for child in (self.left, self.right)
         ) + 1                # Add 1 for this node
```
The animations in this routing should be short or immediate so that it doesn't take much time for a viewer interested in the "macro" behavior of nodes being rotated in the AVL Tree.

**Comment by JMCanning78** [comment]
Created branch Issue289-AVLTree-updateHeight for this work.

**Comment by JMCanning78** [comment]
Closed by PR #299
